src/codeFly/dispatcher/README.md

```diff
@@ -0,0 +1,106 @@
+# Interfaces  
+Here are basic interfaces for front-end team and back-end team to follow:
+
+## Register
+The user is required to input ```username```, ```password```, and ```email address``` on the GUI. Once the user clicks the ```Register``` button, a post request will be sent to the server. Then the server will send a response to indicate if this registration is successful or not in the content body. The user will be navigated to different web pages based on the response.
+```
+Request(font-end)  
+Request Method: POST  
+URI: localhost:8080/register  

```

Comment:
No need to put authority here since we always use resources from our own server.